GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/ContentTypes/govuksectionbreaksettings.config

```diff
@@ -0,0 +1,66 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ContentType Key="9581b375-d5f1-4fcf-affd-9d80883d7812" Alias="govukSectionBreakSettings" Level="2">
+  <Info>
+    <Name>Section break settings</Name>
+    <Icon>icon-remove color-black</Icon>
+    <Thumbnail>folder.png</Thumbnail>
+    <Description></Description>
+    <AllowAtRoot>False</AllowAtRoot>
+    <IsListView>False</IsListView>
+    <Variations>Nothing</Variations>
+    <IsElement>true</IsElement>
+    <HistoryCleanup>
+      <PreventCleanup>False</PreventCleanup>
+      <KeepAllVersionsNewerThanDays></KeepAllVersionsNewerThanDays>
+      <KeepLatestVersionPerDayForDays></KeepLatestVersionPerDayForDays>
+    </HistoryCleanup>
+    <Folder>GOV.UK</Folder>
+    <Compositions />
```

Comment:
You need to add three compositions here (which you do by clicking the 'Compositions' button in the UI for this document type):

1. `Grid (row classes)` which automatically adds support for adding a custom CSS class on the grid row.
2. `Grid (column classes)` which automatically adds support for adding a custom CSS class on the grid column, and importantly allows changing the section break from its default size of two-thirds wide to another size (eg full width) if needed.
3. `CSS classes` which should be on every component and allows adding a custom CSS class to the component, in this case the `hr` element. You'll need to implement yourself in `GovUkSectionBreak.cshtml` but you can find examples in most files in the same folder.